### ISSUE TITLE
Request to be removed as admin

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#3141]
## User story 
As an admin I want to be able to easily request other admins remove me.

## Best solution for this problem
Currently the "remove admin" button is deactivated for yourself, to prevent all admins being removed and leaving no one. You have to get another admin to remove you. Instead of just this message, the button should be "Request to be removed". This send a notification to the other admins, with a button they can click to fulfil your request.

<img width="896" alt="Screen Shot 2020-05-14 at 3 14 40 PM" src="https://user-images.githubusercontent.com/5498878/81888702-0238ed00-95f6-11ea-843a-48dcaf6a5b3b.png">

This is just a "nice to have" since it's possible to just contact another admin outside the system, but I think it would be a better user experience and could be a good bounty project.

**Comment by vickywane** [comment]
Hi, 

Can you explain a bit more on how best to "send a notification to other admin" without making changes to the existing system

**Comment by alanna** [comment]
It could be an email, similar to notifications that go to admins for new expenses, contact form enquiries, etc.

**Comment by vickywane** [comment]
Alright. 

This could be a feature done with serverless functions using existing emailing templates. 

I'd like to wait on input from other core contributors.

**Comment by JoyLubega** [comment]
@znarf How best can I send the notification(Email). Can I send emails with the frontend only, I remember seeing email templates in the API. I beg to be advised

**Comment by Betree** [comment]
This issue introduces some complexity in our codebase. Could we directly allow admins to remove themselves if there's another admin?

**Comment by alanna** [comment]
@Betree yes please! I really just need to be able to remove myself as an admin. This is especially important now because I'm running multiple fiscal hosts with external application forms on their website. I need to be able to create Collectives for them once approved, make the applicant the admin, and then remove myself.

**Comment by Betree** [comment]
Closing as we're going to implement https://github.com/opencollective/opencollective/issues/3303 instead
